ppr-ui/src/components/mhrRegistration/YourHome/ManufacturerMakeModel.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div>
```

Comment:
Not a bad idea to add an id :)

Reply:
Added, however, we already have ids in the parent `YourHome.vue`.